console/network/src/consensus_heights.rs

```diff
@@ -39,10 +75,11 @@ pub const MAINNET_V0_CONSENSUS_VERSION_HEIGHTS: [(ConsensusVersion, u32); 9] = [
     (ConsensusVersion::V7, 7_570_000),
     (ConsensusVersion::V8, 9_430_000),
     (ConsensusVersion::V9, 10_272_000),
+    (ConsensusVersion::V10, 11_114_992),
```

Comment:
Can you make these heights rounded values like the rest of the consensus heights?

Reply:
Apologies for the oversight - I see my AI buddy has some stuff to learn still c8d38ef10

Reply:
Further rounded in - [462994a](https://github.com/ProvableHQ/snarkVM/pull/2828/commits/462994a919c04085c7a0f63764cc6ec33b85b4cc)